packages/@webex/plugin-meetings/src/statsAnalyzer/index.ts

```diff
@@ -938,7 +939,7 @@ export class StatsAnalyzer extends EventsScope {
       this.statsResults[mediaType][sendrecvType].totalPacketsReceived = result.packetsReceived;
 
       if (currentPacketsReceived === 0) {
-        if (receiveSlot) {
+        if (receiveSlot && sourceState === 'live') {
```

Comment:
while you're changing the code here, can you also fix the log from the next line? it shows the `currentPacketsReceived` value at the end, but that value is always going to be zero, it's more useful to show `result.packetsReceived`, because then it will tell us if there were any packets ever received on that slot or not (we will see either zero or some integer).